components/form/CheckboxFormControl.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Checkbox({ id, isChecked = false, isToggle = false, onChange }) {
+  const handleChange = (event) => onChange(event.target.checked);
+
+  return (
+    <input
+      checked={isChecked}
+      className={isToggle ? 'toggle' : 'checkbox'}
+      id={id}
+      name={id}
+      onChange={handleChange}
+      type="checkbox"
+    />
+  );
+}
+
+function CheckboxFormControl({ id, label, isChecked, isToggle, onChange }) {
+  return (
+    <div className="form-control">
+      {label ? (
```

Comment:
Interesting, what's the scenario where `CheckboxFormControl` has no label?

Reply:
you are right - now I cannot think of such situation 🙂 - I will remove that